src/cli/cmds/synth.ts

```diff
@@ -29,6 +43,9 @@ class Command implements yargs.CommandModule {
     const validate = argv.validate;
     const pluginsDir = argv.pluginsDir;
     const reportFile = argv.validationReportsOutputFile;
+    const format = argv.format;
```

Comment:
Shouldn't we validate this can only be `plain` or `helm` somewhere? what is preventing me from passing "bla"?

Reply:
That is a good callout. I thought yargs would take care of it: https://yargs.js.org/docs/#api-reference-choiceskey-choices. But I do not see any doc saying that this would throw an error, would test that. 

But, I believe a custom error would be better for users here. Will add this.